cli/lib/kontena/cli/vpn/create_command.rb

```diff
@@ -54,13 +54,8 @@ def execute
     # @return [String]
     def node_vpn_ip(node)
       return ip unless ip.nil?
-
-      # vagrant
-      if api_url == 'http://192.168.66.100:8080'
-        node['private_ip']
-      else
-        node['public_ip']
-      end
+      
+      node['public_ip'].to_s.empty? ? node['private_ip'].to_s : node['public_ip'].to_s
```

Comment:
I was thinking that we could treat Vagrant as any other "private" env, the user should provide the node and IP. But now thinking that bit further, it's kind of breaking change from user expectation point of view. I'll add the hack back in some form :)